test_runner/regress/test_pageserver_restarts_under_workload.py

```diff
@@ -17,6 +17,12 @@ def test_pageserver_restarts_under_worload(neon_simple_env: NeonEnv, pg_bin: PgB
     n_restarts = 10
     scale = 10
 
+    # the background task may complete the init task delay after finding an
+    # active tenant, but shutdown starts right before Tenant::gc_iteration
+    env.pageserver.allowed_errors.append(
+        r"Gc failed, retrying in \S+: Cannot run GC iteration on inactive tenant"
```

Comment:
This isn't enough, we do `re.match` not search.

Reply:
[8dda5a7](https://github.com/neondatabase/neon/pull/3826/commits/8dda5a784bb38c27c49300d6155f1d384eec1079)